pino.js

```diff
@@ -91,26 +100,35 @@ Pino.prototype.info = genLog(levels.info)
 Pino.prototype.debug = genLog(levels.debug)
 Pino.prototype.trace = genLog(levels.trace)
 
+Object.defineProperty(Pino.prototype, 'levelNum', {
+  get: function getLevelNum () {
+    return this._levelNum
+  },
+  set: function setLevelNum (num) {
+    if (typeof num === 'string') { return this._setLevel(num) }
+    this._levelNum = num
+
+    for (var key in levels) {
+      if (num > levels[key]) {
+        this[key] = noop
+        continue
+      }
```

Comment:
just noticed you remove the check against whether the key has been set to no-op

is there a perf difference to removing this check? we might have to verify perf difference by changing the levels to trigger the prop overwriting

my thought at the time was it's better not to shadow the prototype if possible (in view of the way hidden classes work)